### PR TITLE
tests(build): fix fileExists check, remove chai, run in CI

### PR DESCRIPTION
**Pull request by sidvishnoi**
- Fix fileExists check in `test-build.js` (`stats.fileExists()` is not a function)
- Remove chai dependency, use jasmine's matchers
- Cleanup `builds` dir after running `test:build`
- Run `test:build` in CI

### CODE REVIEW

File: tests/test-build.js [hunk 1]
```diff
@@ -1,35 +1,45 @@
...
     it(`builds the "${profile}" profile and sourcemap`, async () => {
-      expect(await fileExists(profileFile)).to.equal(true);
-      expect(await fileExists(mapFile)).to.equal(true);
+      await expectAsync(fileExists(profileFile)).toBeResolvedTo(true);
+      await expectAsync(fileExists(mapFile)).toBeResolvedTo(true);
     });
     it(`includes sourcemap link for "${profile}"`, async () => {
       const source = await readFile(profileFile, "utf-8");
-      expect(source.includes(`${profile}.js.map`)).to.equal(true);
+      expect(source.includes(`${profile}.js.map`)).toBeTrue();
```
Comment (sidvishnoi) [inline tests/test-build.js:42]: Nit: can use `toContain| here

Comment (sidvishnoi) [inline tests/test-build.js:42]:
```suggestion
      expect(source).toContain(`${profile}.js.map`);
```

File: tests/test-build.js [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
+  const rootDir = path.join(__dirname, "..");
 
   beforeAll(async () => {
     await Promise.all(
       profiles.map(profile => Builder.build({ name: profile }))
     );
   });
 
+  afterAll(() => {
+    execSync("git checkout -- builds", { cwd: rootDir });
```
Comment (saschanaz) [inline tests/test-build.js:32]: `checkout` for this purpose is considered as legacy, how about:

```suggestion
    execSync("git restore builds", { cwd: rootDir });
```

Comment (sidvishnoi) [reply tests/test-build.js:32]: Depends on whether @marcoscaceres has updated his git :smile:

Comment (saschanaz) [reply tests/test-build.js:32]: `sudo login to marcos's computer && sudo apt upgrade everything`

Comment (marcoscaceres) [reply tests/test-build.js:32]: My git is always up to date :)

**Review by marcoscaceres — APPROVED** [review]

**Pull request merged**